rs/consensus/tests/integration.rs

```diff
@@ -40,7 +41,7 @@ fn single_node_is_live() {
 }
 
 #[test]
-fn ecdsa_pubkey_is_produced() -> Result<(), String> {
+fn master_pubkey_is_produced() -> Result<(), String> {
```

Comment:
Maybe 
```suggestion
fn master_pubkeys_are_produced() -> Result<(), String> {
```
and then also
```rust
run_n_rounds_and_check_pubkey[s]
```
below